src/Tribe/Commerce/PayPal/Frontend/Tickets_Form.php

```diff
@@ -57,6 +57,13 @@ public function render() {
 
 		$tickets = $this->main->get_tickets( $post->ID );
 
+		foreach ( $tickets as $key => $ticket ) {
+			/** @var Tribe__Tickets__Ticket_Object $ticket */
+			if ( ! $ticket->date_in_range() ) {
+				unset( $tickets[$key] );
```

Comment:
* :no_entry_sign: There must be a single space after the first square bracket
* :no_entry_sign: There must be a single space before the last square bracket

<sub>hash: 3dab98b81583675c01c47d0fcbee4c0d</sub>